glustercli/cmd/peer.go

```diff
@@ -34,6 +36,8 @@ func init() {
 
 	peerCmd.AddCommand(peerStatusCmd)
 
+        peerListCmd.Flags().StringVar(&flagCmdFilterKey, "key", "", "Filter by metadata key")
```

Comment:
There's something fishy about these indentations or the way github is showing it to me :-/ 

Are you sure all your go files pass gofmt test ?

Reply:
@prashanthpai  Even I was wondering whats wrong, but they seem alright in https://github.com/vpandey-RH/glusterd2/blob/c0dbae250d1b4d905798e09a82f43d0d378a9207/glustercli/cmd/peer.go

Reply:
@prashanthpai  I am  pretty sure it passed the gofmt test. I will look into it though.